var/spack/repos/spack_repo/builtin/packages/lemon/package.py

```diff
@@ -22,6 +22,7 @@ class Lemon(CMakePackage):
     # soplex not mentioned in docs but shown in cmakecache
     # variant("soplex", default=False, description="Enable SOPLEX solver backend") #TODO
 
+    depends_on("c", type="build")  # generated
```

Comment:
Since you are confirming the need for these dependencies, please remove the `# generated` comments here and in the other packages.

Reply:
If you can also confirm the `cxx` dependency, please also remove the comment from those lines as well.

Reply:
it failed without cxx.